### PR TITLE
OSD-7599

### PR DESCRIPTION
**Pull request by robotmaxtron**
Boilerplate update && op-generate to v1

### CODE REVIEW

File: deploy/crds/managed.openshift.io_mustgathers.yaml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
```
Comment (2uasimojo) [inline deploy/crds/managed.openshift.io_mustgathers.yaml:3]: ✓

**Review by 2uasimojo — COMMENTED** [review]
Boilerplate updates are kosher. CRD gen looks like it did the right thing (nit: I would have done that in a separate commit, but no importa).

Watching :eyes:

**Comment by 2uasimojo** [comment]
The `validate` failure makes me think you did `make generate` locally, and your local controller-gen is newer than the one in the bp container. If you run it as `./boilerplate/_lib/container-make generater` it should match what CI expects.

**Comment by csheremeta** [comment]
sorry, removed lgtm comment since this is still marked wip

**Comment by robotmaxtron** [comment]
@2uasimojo Thanks for the tip, I was able to use that build container to generate the crd with the expected version without needing to manage the version on my end.

**Comment by robotmaxtron** [comment]
Thank you for the review @csheremeta, I've removed the WIP label now that the ci tests are passing.

**Comment by 2uasimojo** [comment]
I applied this CRD on a cluster and its conditions are clean

```
[efried@efried hive]$ oc get  customresourcedefinition.apiextensions.k8s.io/mustgathers.managed.openshift.io -o json | jq -r .status.conditions
I0708 14:01:15.057162 1098073 request.go:655] Throttling request took 1.142503187s, request: GET:https://api.efried2.new-installer.openshift.com:6443/apis/snapshot.storage.k8s.io/v1?timeout=32s
[
  {
    "lastTransitionTime": "2021-07-08T19:00:14Z",
    "message": "no conflicts found",
    "reason": "NoConflicts",
    "status": "True",
    "type": "NamesAccepted"
  },
  {
    "lastTransitionTime": "2021-07-08T19:00:14Z",
    "message": "the initial names have been accepted",
    "reason": "InitialNamesAccepted",
    "status": "True",
    "type": "Established"
  }
]
```

Structural schema is now satisfied, and `oc explain` works (cc @dustman9000, xref OSD-7057)

```
[efried@efried hive]$ oc explain mustgathers
I0708 14:02:21.929279 1098415 request.go:655] Throttling request took 1.114178188s, request: GET:https://api.efried2.new-installer.openshift.com:6443/apis/operators.coreos.com/v1?timeout=32s
KIND:     MustGather
VERSION:  managed.openshift.io/v1alpha1

DESCRIPTION:
     MustGather is the Schema for the mustgathers API

FIELDS:
   apiVersion	<string>
     APIVersion defines the versioned schema of this representation of an
     object. Servers should convert recognized schemas to the latest internal
     value, and may reject unrecognized values. More info:
     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources

   kind	<string>
     Kind is a string value representing the REST resource this object
     represents. Servers may infer this from the endpoint the client submits
     requests to. Cannot be updated. In CamelCase. More info:
     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds

   metadata	<Object>
     Standard object's metadata. More info:
     https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata

   spec	<Object>
     MustGatherSpec defines the desired state of MustGather

   status	<Object>
     MustGatherStatus defines the observed state of MustGather

[efried@efried hive]$ oc explain mustgathers.spec
I0708 14:02:34.666643 1098470 request.go:655] Throttling request took 1.000682053s, request: GET:https://api.efried2.new-installer.openshift.com:6443/apis/admissionregistration.k8s.io/v1beta1?timeout=32s
KIND:     MustGather
VERSION:  managed.openshift.io/v1alpha1

RESOURCE: spec <Object>

DESCRIPTION:
     MustGatherSpec defines the desired state of MustGather

FIELDS:
   caseID	<string> -required-
     The is of the case this must gather will be uploaded to

   caseManagementAccountSecretRef	<Object> -required-
     the secret container a username and password field to be used to
     authenticate with red hat case management systems

   mustGatherImages	<[]string>
     The list of must gather images to run, optional, it will default to:
     $DEFAULT_MUST_GATHER_IMAGE

   proxyConfig	<Object>
     This represents the proxy configuration to be used. If left empty it will
     default to the cluster-level proxy configuration.

   serviceAccountRef	<Object>
     the service account to use to run the must gather job pod, defaults to
     default +kubebuilder:default:="{Name:default}"
```

**Pull request merged**